src/deluge/gui/ui/keyboard/layout/column_controls.cpp

```diff
@@ -27,6 +27,7 @@
 #include "processing/sound/sound_instrument.h"
 #include "storage/flash_storage.h"
 #include "util/functions.h"
+#include "util/lookuptables/lookuptables.h"
```

Comment:
is this used? I don't see an addition that touches any tables

Reply:
MAX_CHORD_NOTES is defined in lookuptables.h and used here